src/Notofications/Notifications.API/appsettings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "RabbitMq": {
+    "RabbitMqHost": "localhost",
+    "RabbitMqQueue": "MyLocalQueue"
+  },
+  //TODO: dont forgot to change this settings before sending emails
```

Comment:
?

Reply:
dont know where to store password and dont want to save it on pc os in keys

Reply:
thats why change it myself if need notification service